components/host-dashboard/reports-section/TotalMoneyManagedHistorical.js

```diff
@@ -93,6 +94,21 @@ const getChartOptions = intl => ({
       (_, idx) => `${intl.formatDate(new Date(0, idx), { month: 'short' }).toUpperCase()}`,
     ),
   },
+  yaxis: {
+    labels: {
+      minWidth: 38,
+      formatter: function (value) {
+        return value < 1000 ? value : `${Math.round(value / 1000)}k`;
```

Comment:
Not sure about the k notation. @Betree ?

Reply:
Round the value, make sure to not display the cents. Then use formatCurrency to have the separators like in the tooltip.

Reply:
We use the `k` values int he host fees section; this is why I opted for this `k` notation btw just to make things consistent. If we are going with rounding and not using the `k` notation maybe we should do it for both sections right? 🤔

Reply:
Yes, I was not aware that it was used in the other graph. Everything should be consistent of course. @Betree will confirm.

Reply:
We implemented the 1st graph to match the design, and that is what they used. I personally like it because we're going to display large numbers in there. We should definitely have separators, so maybe we rather want something like this in the `else` part:

```suggestion
        return value < 1000 ? value : `${Math.round(value / 1000).toLocaleString()}k`;
```

Let's follow up with that somewhere else to have a consistent update across all charts.

Reply:
I've incorporated this suggestions so that we have the separators. 👍🏽

Reply:
@SudharakaP Can we make sure we have something consistent between all the graphs? Maybe introduce a common config or helpers?

Reply:
I've added a common helper for this so that both charts y-axis labels are rendered in the same way.